sycl/CMakeLists.txt

```diff
@@ -71,130 +71,23 @@ set(SYCL_INCLUDE_DIR "include")
 set(SYCL_INCLUDE_BUILD_DIR ${LLVM_BINARY_DIR}/${SYCL_INCLUDE_DIR})
 set(SYCL_INCLUDE_DEPLOY_DIR ${CMAKE_INSTALL_PREFIX}/${SYCL_INCLUDE_DIR})
 
-# Find OpenCL headers and libraries installed in the system and use them to
-# build SYCL runtime.
-# WARNING: use with caution, building SYCL runtime with OpenCL implementation
-# instead of Khronos ICD loader might cause build and/or portability issues.
-option(OpenCL_BUILD_WITH_SYSTEM_SDK "Build SYCL with OpenCL coming from the build system" OFF)
-
-if( (OpenCL_INCLUDE_DIR AND OpenCL_LIBRARY) OR OpenCL_BUILD_WITH_SYSTEM_SDK)
-  find_package(OpenCL)
```

Comment:
It looks like new cmake does not allow to use system OpenCL ICD loader/ Header. Someone may use it and there will be a regression in their case.

Reply:
I think it is broken, if configure.py is our recommended entry point at least. See my comment above.
Can you confirm or maybe I'm missing how to use that?
If it's broken already I would just keep it officially dead.

Reply:
There is mess with these variables (you can see description in https://cmake.org/cmake/help/v3.1/module/FindOpenCL.html)
As for my understanding:
 - OpenCL_*s are INPUTparameters which are used by FindOpenCL module (parameters in single mode are ignored). 
 - OpenCL_* are OUTPUT parameters which are set by  FindOpenCL module (parameters in plural mode should not be used after). 
It should work if used properly (I fixed it a few times in the past).

Reply:
Agreed with @vladimirlaz to kill this completely. Doesn't work now. If we ever need this case again will restore separately. I'll cleanup code in the next iteration.